[{"url":"https://api.github.com/repos/elastic/elasticsearch/issues/comments/340053106","html_url":"https://github.com/elastic/elasticsearch/issues/27148#issuecomment-340053106","issue_url":"https://api.github.com/repos/elastic/elasticsearch/issues/27148","id":340053106,"node_id":"MDEyOklzc3VlQ29tbWVudDM0MDA1MzEwNg==","user":{"login":"morphers82","id":1844753,"node_id":"MDQ6VXNlcjE4NDQ3NTM=","avatar_url":"https://avatars0.githubusercontent.com/u/1844753?v=4","gravatar_id":"","url":"https://api.github.com/users/morphers82","html_url":"https://github.com/morphers82","followers_url":"https://api.github.com/users/morphers82/followers","following_url":"https://api.github.com/users/morphers82/following{/other_user}","gists_url":"https://api.github.com/users/morphers82/gists{/gist_id}","starred_url":"https://api.github.com/users/morphers82/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/morphers82/subscriptions","organizations_url":"https://api.github.com/users/morphers82/orgs","repos_url":"https://api.github.com/users/morphers82/repos","events_url":"https://api.github.com/users/morphers82/events{/privacy}","received_events_url":"https://api.github.com/users/morphers82/received_events","type":"User","site_admin":false},"created_at":"2017-10-27T18:43:01Z","updated_at":"2017-10-27T18:43:01Z","author_association":"NONE","body":"I had a problem like this using php client.  It was because I didn't include the upsert field while doing a scripted_upsert","performed_via_github_app":null},{"url":"https://api.github.com/repos/elastic/elasticsearch/issues/comments/340074605","html_url":"https://github.com/elastic/elasticsearch/issues/27148#issuecomment-340074605","issue_url":"https://api.github.com/repos/elastic/elasticsearch/issues/27148","id":340074605,"node_id":"MDEyOklzc3VlQ29tbWVudDM0MDA3NDYwNQ==","user":{"login":"rfraposa","id":5323324,"node_id":"MDQ6VXNlcjUzMjMzMjQ=","avatar_url":"https://avatars1.githubusercontent.com/u/5323324?v=4","gravatar_id":"","url":"https://api.github.com/users/rfraposa","html_url":"https://github.com/rfraposa","followers_url":"https://api.github.com/users/rfraposa/followers","following_url":"https://api.github.com/users/rfraposa/following{/other_user}","gists_url":"https://api.github.com/users/rfraposa/gists{/gist_id}","starred_url":"https://api.github.com/users/rfraposa/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/rfraposa/subscriptions","organizations_url":"https://api.github.com/users/rfraposa/orgs","repos_url":"https://api.github.com/users/rfraposa/repos","events_url":"https://api.github.com/users/rfraposa/events{/privacy}","received_events_url":"https://api.github.com/users/rfraposa/received_events","type":"User","site_admin":false},"created_at":"2017-10-27T20:15:49Z","updated_at":"2017-10-27T20:16:19Z","author_association":"NONE","body":"I am using Logstash with the following output - which includes the `upsert` parameter:\r\n\r\n```\r\noutput {\r\n\telasticsearch {\r\n\t\taction => \"update\"\r\n \t\tindex => \"my_destination\"\r\n\t\tdocument_type => \"doc\"\r\n\t\tdocument_id => \"%{trip_id}\"\r\n\t\tscripted_upsert => true\r\n\t\tscript => \"update_my_docs\"\r\n\t\tscript_type => \"indexed\"\r\n\t\tscript_lang => \"painless\"\r\n\t\tscript_var_name => \"event\"\r\n\t\tupsert => \"null\"\r\n\t}\r\n}\r\n```\r\n\r\nIt is certainly possible that this is a bug with Logstash. ","performed_via_github_app":null},{"url":"https://api.github.com/repos/elastic/elasticsearch/issues/comments/340099329","html_url":"https://github.com/elastic/elasticsearch/issues/27148#issuecomment-340099329","issue_url":"https://api.github.com/repos/elastic/elasticsearch/issues/27148","id":340099329,"node_id":"MDEyOklzc3VlQ29tbWVudDM0MDA5OTMyOQ==","user":{"login":"rfraposa","id":5323324,"node_id":"MDQ6VXNlcjUzMjMzMjQ=","avatar_url":"https://avatars1.githubusercontent.com/u/5323324?v=4","gravatar_id":"","url":"https://api.github.com/users/rfraposa","html_url":"https://github.com/rfraposa","followers_url":"https://api.github.com/users/rfraposa/followers","following_url":"https://api.github.com/users/rfraposa/following{/other_user}","gists_url":"https://api.github.com/users/rfraposa/gists{/gist_id}","starred_url":"https://api.github.com/users/rfraposa/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/rfraposa/subscriptions","organizations_url":"https://api.github.com/users/rfraposa/orgs","repos_url":"https://api.github.com/users/rfraposa/repos","events_url":"https://api.github.com/users/rfraposa/events{/privacy}","received_events_url":"https://api.github.com/users/rfraposa/received_events","type":"User","site_admin":false},"created_at":"2017-10-27T21:43:50Z","updated_at":"2017-10-27T21:43:50Z","author_association":"NONE","body":"I spoke with @andrewvc on the Logstash team and they found a small bug in the bulk upsert request sent by LS. Closing this issue for now, in hopes that the fix will resolve this issue.","performed_via_github_app":null}]